go.mod

```diff
@@ -17,6 +18,7 @@ require (
 	github.com/google/go-github/v33 v33.0.0
 	github.com/google/gofuzz v1.2.0
 	github.com/gosuri/uitable v0.0.4
+	github.com/kinvolk/container-linux-config-transpiler v0.9.1
```

Comment:
We still need to tag it, but this is now `github.com/flatcar-linux/container-linux-config-transpiler`.

Reply:
Waiting for https://github.com/flatcar-linux/container-linux-config-transpiler/pull/14 to be merged, then we can release `v0.9.2`.

Reply:
Fixed.